### ISSUE TITLE
Gestures on UITableView inside custom view controller not recognized [incl. solution]

### ISSUE THREAD

**Issue by caloon** [Orderella/PopupDialog#117]
# Report
Sorry, but I'm not gonna fill out everything, because I already found the solution and the issue is one applicable to all environments.

## What did you do?
Add a custom view controller to PopupDialog containing a UITableView

## What did you expect to happen?
Table view should work as expected

## What happened instead?
Table view could not detect any gestures (i.e. selection). 

I already found the solution. Adding these lines:
```
tapRecognizer.cancelsTouchesInView = false             
panRecognizer.cancelsTouchesInView = false
```
to PopupDialog.swift solves the issue. 

I'm not familiar with opening pull requests, so I'm opening an issue instead.

**Comment by mwfire** [comment]
Hey @caloon, thanks for your report. This seems to be a duplicate of #107. The implementation will be released with the next version and is ready for testing on develop: pod 'PopupDialog', :git => 'https://github.com/Orderella/PopupDialog.git', :branch => 'development'

Tbh, I am not entirely sure about canceling the panRecognizer touches. Can you confirm this won't break any scrolling functionality with table views?

Thanks!

**Comment by caloon** [comment]
Only tested in simulator (Xcode 8.2, iOS 10.0) so far, but scrolling seemed to work fine.

**Comment by mwfire** [comment]
Released with #163
